src/components/PreviousReflections.svelte

```diff
@@ -161,6 +158,12 @@
 
             <Lifewheel data={currentReflection.data} {tweenedLifewheel} class="max-w-sm" />
 
+            {#if currentReflection.comment != null && currentReflection.comment.length > 0}
+                <CommentView textData={currentReflection.comment} />
+            {:else}
+                <CommentView textData="." hidden />
```

Comment:
This is a bit of a hacky solution to get a non-empty string to appear in the component so it gets the same height as reflections with comments. I've set the visibility to hidden so that when scrolling between reflections the page layout remains consistent. This solution can surely be improved though.